src/components/Composer/index.tsx

```diff
@@ -364,8 +288,8 @@ function Composer({
         const unsubscribeFocus = navigation.addListener('focus', () => document.addEventListener('paste', handlePaste));
         const unsubscribeBlur = navigation.addListener('blur', () => document.removeEventListener('paste', handlePaste));
 
-        if (_.isFunction(forwardedRef)) {
-            forwardedRef(textInput.current);
+        if (typeof ref === 'function' && textInput.current) {
```

Comment:
Any reason for adding ` && textInput.current` condition here?

Reply:
Not really, removed!